src/scss/components/_post.scss

```diff
@@ -86,100 +48,68 @@
 
     &::after {
       @include tools.position(0 0 0 0);
+      @include tools.z-index('bump');
       content: '';
       display: block;
     }
   }
-
-  .byline,
-  .next-event {
-    font-size: var(--xsmall);
-  }
-
-  .next-event {
-    padding-left: 1em;
-  }
 }
 
-.card-inner {
-  background-color: var(--card, var(--callout));
-  clip-path: var(--crop);
-  padding: var(--gutter);
+[data-post='large'] {
+  --post-margin: minmax(0, 1fr);
 }
 
-// Features
-[data-card='hero'] {
-  display: grid;
-  grid-template-columns:
-    minmax(0, 1fr) [main-start] minmax(min-content, var(--page))
-    [main-end] minmax(0, 1fr);
-
-  > * {
-    grid-column: var(--area, main);
-  }
-}
+// Elements
+// --------
 
-[data-feature='small'] {
-  --card: var(--bg);
-  --rotate: -3deg;
-  --post-type: var(--bg);
-  --post-type-active: var(--callout);
-  background-color: var(--feature-accent);
-  background-image: linear-gradient(to bottom, transparent, var(--feature));
-  margin: calc(var(--half-shim) * -1);
-
-  [data-card] {
-    --shadow: transparent;
-    margin: var(--quarter-shim);
-  }
+[data-hero='post'] {
+  grid-area: hero;
 }
 
-[data-feature='large'] {
-  font-size: 1rem;
-  grid-column: 1 / -1;
-  margin: var(--gutter) 0;
-}
+.type-flag {
+  @include tools.font-family('alt');
+  background: var(--highlight);
+  color: var(--bg);
+  font-size: var(--xsmall);
+  grid-area: type;
+  place-self: start end;
```

Comment:
whoa, I haven't seen this yet. <googling it now>

Reply:
`place-*` is the shorthand for `align-*` and `justify-*` 😄